### PR TITLE
Maywood: Remove code which targets Gutenberg UI

### LINKED ISSUE #47117: Editor: When a color palette is chosen for the site, poor-contrast color changes happen in the UI
When a custom color palette is loaded on a site, the `:hover` and `:focus` state of UI elements in Gutenberg claim the primary color of the site as the background color, while the text color remains black. This creates a poor contrast and horrible experience while using the editor:

Screen Shot 2020-11-02 at 21 56 14

Screen Shot 2020-11-02 at 21 55 30

Screen Shot 2020-11-02 at 21 57 29

It seems to be isolated to Varia-child themes:

Not reproducible: Spearhead, Twenty Twenty, Radcliffe 2, Seedlet
Reproducible: All Varia-child

#### Steps to reproduce

1. Starting at URL: https://wordpress.com
2. Activate a Varia-child theme in Design > Themes
3. Design > Customize > Colors & Backgrounds, pick a custom color palette, "Save changes"
4. Open a post in the editor

#### What I expected

I expected the UI elements to retain the normal Gutenberg UI color palette as seen below with the theme Seedlet, for example:

Screen Shot 2020-11-04 at 13 39 48

#### Context / Source

#dogfooding

### PR DESCRIPTION
**Pull request by scruffian**
This was added here: https://github.com/Automattic/themes/commit/69ca43eca44cc88a0495a2f903be03335e73c974

But it causes this issue: https://github.com/Automattic/themes/issues/2855

So I think we should remove it.

Fixes https://github.com/Automattic/wp-calypso/issues/47117

### CODE REVIEW

**Review by jffng — COMMENTED** [review]
This works in my testing — the block toolbar no longer inherits the custom text color.

I'm having trouble identifying what the actual code changes were though because of the automating linting correction.

File: maywood/inc/wpcom-editor-colors.php [hunk 1]
```diff
@@ -164,75 +237,114 @@
 			#editor .editor-styles-wrapper .wp-block-pullquote,
 			#editor .editor-styles-wrapper .wp-block-code > code,
 			#editor .editor-styles-wrapper .wp-block-verse,
-			#editor .editor-styles-wrapper .components-button,
```
Comment (scruffian) [inline maywood/inc/wpcom-editor-colors.php:167]: @jffng This is the only change

**Comment by jffng** [comment]
Committed in r57404.

**Comment by scruffian** [comment]
Thank you

**Pull request merged**